app/models/warpable.rb

```diff
@@ -18,6 +18,9 @@ class Warpable < ActiveRecord::Base
   belongs_to :map
   belongs_to :user
 
+  has_paper_trail on: %i(create update), only: %i(nodes),
+                  unless: proc { |t| t.count_version == 1 }
```

Comment:
Due to this condition, version is not created when ```count_version==1``` 
It is created only when ```count_version==0``` which is only in the case of update. 
```suggestion
                  unless: proc { |t| t.count_version == 1 }
```
But if it's happening that's wierd.